packages/jellyfish-api-core/src/category/evm.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiClient, BigNumber } from '../.'
+
+/**
+ * EVM RPCs for DeFi Blockchain
+ */
+export class Evm {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Creates an EVM transaction submitted to local node and network
+   * @param {string} from
+   * @param {number} nonce
+   * @param {number} gasPrice
+   * @param {number} gasLimit
+   * @param {string} to
+   * @param {BigNumber} value
+   * @param {string} data
```

Comment:
optional param in jsdoc
```suggestion
   * @param {string} [data]
```